book/src/developer-guides/collator-set-up-guide.md

```diff
@@ -0,0 +1,218 @@
+![picasso-bannar](./picasso-bannar.png)
+
+# COLLATOR SET UP GUIDE
+
+In this document we will cover how to set up a collaborator with Composable 
+Finance. There are several ways to do that.
+
+* Build node locally
+
+* Download pre-build node from GitHub releases
+
+* Run node in docker
+
+## Build from Source
+
+In this step we will set up a rust compiler, toolchain and build a node.
+
+### Setup required libraries
+
+```sh
+sudo apt install -y git clang curl libssl-dev llvm libudev-dev
+```
+
+### Setup Rust binary and Toolchain
+
+```sh
+#!/bin/bash
+
+RUST_C="nightly-2021-11-07"
+
+curl https://sh.rustup.rs -sSf | sh -s -- -y && \
+export PATH="$PATH:$HOME/.cargo/bin" && \
+rustup toolchain uninstall $(rustup toolchain list) && \
+rustup toolchain install $RUST_C && \
+rustup target add wasm32-unknown-unknown --toolchain $RUST_C && \
+rustup default $RUST_C && \
+rustup show
+```
+
+### Get project and build node
+
+```sh
+git clone --depth 1 --branch v2.1.3 https://github.com/ComposableFi/composable.git && \
```

Comment:
the last release has a tag [v2.1.6](https://github.com/ComposableFi/composable/releases/tag/v2.1.6)